src/Card.tsx

```diff
@@ -0,0 +1,176 @@
+                    grey === true && fr.cx("fr-card--grey"),
+                    iconId !== undefined && fr.cx(iconId),
+                    classes.root,
+                    className
+                )}
+                ref={ref}
+            >
+                <div className={cx(fr.cx("fr-card__body"), classes.body)}>
+                    <div className={cx(fr.cx("fr-card__content"), classes.content)}>
+                        <h3 className={cx(fr.cx("fr-card__title"), classes.title)}>
+                            <Link {...linkProps} className={cx(classes.link)}>
+                                {title}
+                            </Link>
+                        </h3>
+                        {desc !== undefined && (
+                            <p className={cx(fr.cx("fr-card__desc"), classes.desc)}>{desc}</p>
+                        )}
+                        <div className={cx(fr.cx("fr-card__start"), classes.start)}>
+                            {start}
+                            {detail !== undefined && (
+                                <p className={cx(fr.cx("fr-card__detail"), classes.detail)}>
+                                    {detail}
+                                </p>
+                            )}
+                        </div>
+                        <div className={cx(fr.cx("fr-card__end"), classes.end)}>
+                            {end}
+                            {endDetail !== undefined && (
+                                <p className={cx(fr.cx("fr-card__detail"), classes.endDetail)}>
+                                    {endDetail}
+                                </p>
+                            )}
+                        </div>
+                    </div>
+                    {footer !== undefined && (
+                        <p className={cx(fr.cx("fr-card__footer"), classes.footer)}>{footer}</p>
+                    )}
+                </div>
+                {/* ensure we dont have an empty imageUrl string */}
+                {(imageUrl !== undefined && imageUrl.length && (
+                    <div className={cx(fr.cx("fr-card__header"), classes.header)}>
+                        <div className={cx(fr.cx("fr-card__img"), classes.img)}>
+                            <img
+                                className={cx(fr.cx("fr-responsive-img"), classes.imgTag)}
+                                src={imageUrl}
+                                alt={imageAlt}
+                            />
+                        </div>
+                        {(badges !== undefined && badges.length && (
+                            <ul className={cx(fr.cx("fr-badges-group"), classes.badges)}>
+                                {badges.map((badge, i) => (
+                                    <li key={i}>{badge}</li>
+                                ))}
+                            </ul>
+                        )) ||
+                            null}
```

Comment:
```suggestion
                        ))}
```
Ther's no need for a fallback to `null`, `false` is a falid `ReactNode` that is equivalent to `null`.